.circleci/config.yml

```diff
@@ -338,9 +339,9 @@ workflows:
       - deploy-to-ecs:
           requires:
             - docker-push
-          filters:
-            branches:
-              only: /^release-1\.17.\d+$/
+#          filters:
+#            branches:
+#              only: /^release-2\.0.\d+$/
```

Comment:
@griggheo looking at this regex, it appears that it may not find branches with a pre-release tag e.g. `-rc.1` or `-beta.1`